neurodsp/filt/checks.py

```diff
@@ -164,18 +170,25 @@ def check_filter_properties(filter_coefs, a_vals, fs, pass_type, f_range,
 
     # Compute pass & transition bandwidth
     pass_bw = compute_pass_band(fs, pass_type, f_range)
-    transition_bw = compute_transition_band(f_db, db, transitions[0], transitions[1])
+    transition_bw, f_range_trans = compute_transition_band(f_db, db, transitions[0], transitions[1],
+                                                           return_freqs=True)
 
     # Raise warning if transition bandwidth is too high
     if transition_bw > pass_bw:
         passes = False
         warn('Transition bandwidth is  {:.1f}  Hz. This is greater than the desired'\
              'pass/stop bandwidth of  {:.1f} Hz'.format(transition_bw, pass_bw))
 
-    # Print out transition bandwidth and pass bandwidth to the user
+    # Report filter properties
+    if verbose or save_properties:
+        filt_report = gen_filt_report(pass_type, filt_type, fs, f_db, db, pass_bw,
+                                      transition_bw, f_range, f_range_trans)
+
     if verbose:
-        print('Transition bandwidth is {:.1f} Hz.'.format(transition_bw))
```

Comment:
I think, from the perspective of the `print_transitions` that is called from in the overlying filter functions, this shouldn't change (it should still just print the transitions, rather than the whole filter report). 
We could change the name here (verbose -> print_transitions) for consistency. 
This is technically breaking, but it's an internal function, so I think we can get away with it.

Reply:
I reverted this back to how it was.